examples/bdaq/test/test_bdaq53_eth.py

```diff
@@ -68,23 +71,22 @@ def test(self):
                 print(tick)
                 tick_old = tick
 
-            if doprint==True:
+            if doprint is True:
```

Comment:
I am not sure about all those `==` to `is` change see: https://stackoverflow.com/questions/15008380/double-equals-vs-is-in-python "1==True returns True, but 1 is True returns False"

Reply:
Fixed